software/23.Elixir/README.md

```diff
@@ -0,0 +1,22 @@
+# Workshop 23 - Build a Chat service using Elixir and Phoenix
+
+In this Workshop, you will learn:
+ - The basics of [Elixir](https://elixir-lang.org/)
+ - How to create a simple web app using [Phoenix](https://www.phoenixframework.org/)
+ - Understand how to build simple [live views](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html)
+
+## Step 0 - Setup
+
+All the required information to start this workshop can be found in [SETUP.md](./SETUP.md)
+
+## Step 1 - Once upon a time...
+
+there was a little developper (let's call him Little Timmy) who wanted to communicate with his friends.
+Rather than using Discord, or some social media, or even IRC, this developper wanted to build his chat app himself...
+Of course, Little Timmy looked on Ycmbinator what was the hot JS framework these days, but scrolling through HackerNews,
+Little Timmy saw an article about a new way of doing distributed web apps, and thought he could maybe give it a try.
+Let's help him build the chat app of his dreams ! :)~
```

Comment:
You could maybe called this step introduction because studants don't need to do anything and this step provide a context. Otherwise, explanation the context with Timmy is very cool because this allows us to project ourselves on a use case.